meson.build

```diff
@@ -309,6 +281,7 @@ edbg = executable(
 	 'ecmd/ecmd-core/cmd/ecmdPowerUser.C',
 	 'ecmd/ecmd-core/cmd/ecmdPsiUser.C',
 	 'ecmd/ecmd-core/cmd/ecmdPnorUser.C',
+	 'ecmd/ecmd-core/cmd/ecmdSpiUser.C',
```

Comment:
You should not have to add this to the build if you have the defines setup right

Reply:
/esw/san5/devenrao/sdk1205_p10/sysroots/x86_64-oesdk-linux/usr/libexec/arm-openbmc-linux-gnueabi/gcc/arm-openbmc-linux-gnueabi/9.2.0/real-ld: edbg@exe/ecmd_ecmd-core_cmd_ecmdInterpreter.C.o: in function `ecmdCommandInterpreter(int, char**)':
/esw/san5/lkammath/EDBG-ISTEP-PR/ecmd-pdbg/build/../ecmd/ecmd-core/cmd/ecmdInterpreter.C:107: undefined reference to `ecmdBiasVoltageUser(int, char**)'
/esw/san5/devenrao/sdk1205_p10/sysroots/x86_64-oesdk-linux/usr/libexec/arm-openbmc-linux-gnueabi/gcc/arm-openbmc-linux-gnueabi/9.2.0/real-ld: /esw/san5/lkammath/EDBG-ISTEP-PR/ecmd-pdbg/build/../ecmd/ecmd-core/cmd/ecmdInterpreter.C:177: undefined reference to `ecmdFruPowerUser(int, char**)'
/esw/san5/devenrao/sdk1205_p10/sysroots/x86_64-oesdk-linux/usr/libexec/arm-openbmc-linux-gnueabi/gcc/arm-openbmc-linux-gnueabi/9.2.0/real-ld: /esw/san5/lkammath/EDBG-ISTEP-PR/ecmd-pdbg/build/../ecmd/ecmd-core/cmd/ecmdInterpreter.C:456: undefined reference to `ecmdPutSpiUser(int, char**)'
/esw/san5/devenrao/sdk1205_p10/sysroots/x86_64-oesdk-linux/usr/libexec/arm-openbmc-linux-gnueabi/gcc/arm-openbmc-linux-gnueabi/9.2.0/real-ld: /esw/san5/lkammath/EDBG-ISTEP-PR/ecmd-pdbg/build/../ecmd/ecmd-core/cmd/ecmdInterpreter.C:474: undefined reference to `ecmdQueryBiasStateUser(int, char**)'
/esw/san5/devenrao/sdk1205_p10/sysroots/x86_64-oesdk-linux/usr/libexec/arm-openbmc-linux-gnueabi/gcc/arm-openbmc-linux-gnueabi/9.2.0/real-ld: /esw/san5/lkammath/EDBG-ISTEP-PR/ecmd-pdbg/build/../ecmd/ecmd-core/cmd/ecmdInterpreter.C:296: undefined reference to `ecmdGetSpiUser(int, char**)'
/esw/san5/devenrao/sdk1205_p10/sysroots/x86_64-oesdk-linux/usr/libexec/arm-openbmc-linux-gnueabi/gcc/arm-openbmc-linux-gnueabi/9.2.0/real-ld: /esw/san5/lkammath/EDBG-ISTEP-PR/ecmd-pdbg/build/../ecmd/ecmd-core/cmd/ecmdInterpreter.C:366: undefined reference to `ecmdAdalPsiUser(int, char**)'
/esw/san5/devenrao/sdk1205_p10/sysroots/x86_64-oesdk-linux/usr/libexec/arm-openbmc-linux-gnueabi/gcc/arm-openbmc-linux-gnueabi/9.2.0/real-ld: /esw/san5/lkammath/EDBG-ISTEP-PR/ecmd-pdbg/build/../ecmd/ecmd-core/cmd/ecmdInterpreter.C:525: undefined reference to `ecmdSystemPowerUser(int, char**)'
/esw/san5/devenrao/sdk1205_p10/sysroots/x86_64-oesdk-linux/usr/libexec/arm-openbmc-linux-gnueabi/gcc/arm-openbmc-linux-gnueabi/9.2.0/real-ld: /esw/san5/lkammath/EDBG-ISTEP-PR/ecmd-pdbg/build/../ecmd/ecmd-core/cmd/ecmdInterpreter.C:292: undefined reference to `ecmdGetPnorUser(int, char**)'
/esw/san5/devenrao/sdk1205_p10/sysroots/x86_64-oesdk-linux/usr/libexec/arm-openbmc-linux-gnueabi/gcc/arm-openbmc-linux-gnueabi/9.2.0/real-ld: /esw/san5/lkammath/EDBG-ISTEP-PR/ecmd-pdbg/build/../ecmd/ecmd-core/cmd/ecmdInterpreter.C:452: undefined reference to `ecmdPutPnorUser(int, char**)'
/esw/san5/devenrao/sdk1205_p10/sysroots/x86_64-oesdk-linux/usr/libexec/arm-openbmc-linux-gnueabi/gcc/arm-openbmc-linux-gnueabi/9.2.0/real-ld: /esw/san5/lkammath/EDBG-ISTEP-PR/ecmd-pdbg/build/../ecmd/ecmd-core/cmd/ecmdInterpreter.C:448: undefined reference to `ecmdPowerModeUser(int, char**)'
collect2: error: ld returned 1 exit status

Hitting this if I don't compile them as ecmdInterpreter uses it. but all the defines aren't like ECMD_REMOVE_POWER_FUNCTIONS is not commented out

Reply:
This is your problem to figure out.  It works properly in the build process I support.  This is why I don't want two parallel build processes.

Reply:
Yes. I will check it & send another patch once I figure out. but, right now things look fine and istep control is working fine . Do you have any other comments?

Reply:
fixed